### DISCUSSION TITLE
Updates and modernizations ideas

### DISCUSSION THREAD

**Discussion by kaosine** [pulsar-edit/.github#37] [Pulsar Features]
Since we're bringing this up to date and I've also suggested it on our upstream master Ive had a few ideas I'm sharing here as well:

1. Naming schemes have changed since master was the default, I propose changing master to main or some other form of this. 
2. Clean up the tags into releases if possible, while making the new tags x.y.z in such a form that if it's <1.0 tag it as 0.x, >1 tag as 1.<current number here>.y. Just for clean up purposes to make things clearer going forward. Especially since the UI in github prioritizes releases over tags in showing that a recent release has happened. That way it's easier to know what has been touched recently and what hasn't beyond just commit dates. Since those aren't always reliable on determining if the project is still active.
3. Treat Atom-community as our upstream rhel counterpart. Any changes we make that get things "stable" are to be immediately discussed and upstreamed with them. 

Considering we're moving forward with these things going forward, quite a few of these will be org wide and tie in to general cleanup and modernization, these seem like good things to keep in mind. ***ESPECIALLY*** since we are taking our current efforts to clean up and archive old packages, these should be applied only to the ones we choose to maintain.

**Comment by ghost** [comment]
> Naming schemes have changed since master was the default, I propose changing master to main or some other form of this.

You mentioned in the Discord that it had many dislikes.
